# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/uclibc/xtensa.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- uClibc-ng-1.0.38/extra/Configs/defconfigs/xtensa/defconfig.vanilla	2021-05-01 13:34:41.790757632 +0200
+++ uClibc-ng-1.0.38/extra/Configs/defconfigs/xtensa/defconfig	2021-05-01 13:34:20.709757641 +0200
@@ -0,0 +1 @@
+TARGET_xtensa=y
